### PR TITLE
User address on subscription details base

### PR DESCRIPTION
**Pull request by grpse**
### Why

Added new function user_details to get users information as json. There is some checks about user requesting address of user to test if this user is contributing to some of requester projects.

### Wrap up checklist

- [X] All new code has tests
- [X] Comments's added to columns / views / functions ([ADD COMMENT](https://www.postgresql.org/docs/9.4/static/sql-comment.html)...) 
- [X] Code is documented on docs. link PR from [docs repo](https://github.com/common-group/common)
- [x] Code is reviewed

### CODE REVIEW

File: migrations/2018-06-05-130001_user_details_on_community_api/up.sql [hunk 1]
```diff
@@ -0,0 +1,60 @@
+CREATE OR REPLACE FUNCTION community_service_api.user_details(user_id uuid)
```
Comment (devton) [inline migrations/2018-06-05-130001_user_details_on_community_api/up.sql]: lets use only id instead user_id, theses params are mapped on request

File: migrations/2018-06-05-130001_user_details_on_community_api/up.sql [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+                u.key
+            from
+                community_service.users as u
+            where
+                u.id = _user_id
+            limit 1
+            into _user;
+                
+            select json_build_object(
+                'id', _user.id,
```
Comment (devton) [inline migrations/2018-06-05-130001_user_details_on_community_api/up.sql:48]: we can return the external_id too

File: specs/sql-specs/community-service-api/function.user_details_test.sql [hunk 3]
```diff
@@ -0,0 +1,29 @@
+BEGIN;
+    -- insert seed data for basic user/platform/project/reward
+    \i /specs/sql-support/insert_platform_user_project.sql
+    \i /specs/sql-support/clean_sets_helpers.sql
+
+    select plan(1);
+
+    create or replace function test_access_with_scoped_to_user_details()
```
Comment (devton) [inline specs/sql-specs/community-service-api/function.user_details_test.sql]: add more tests for platform_user role and anonymous role

**Review by devton — CHANGES_REQUESTED** [review]

**Comment by grpse** [comment]
all changes made and docs added (i made a PR to common docs)

**Review by devton — APPROVED** [review]

**Pull request merged**